test/e2e/run-all.sh

```diff
@@ -51,6 +50,7 @@ concurrently --kill-others \
   'npm run sendwithprivatedapp' \
   'sleep 5 && mocha test/e2e/incremental-security.spec'
 
+export GANACHE_ARGS="$GANACHE_ARGS --deterministic --account=0x53CB0AB5226EEBF4D872113D98332C1555DC304443BEE1CF759D15798D3C55A9,25000000000000000000"
```

Comment:
The `$GANACHE_ARGS` here includes all of the previous exported values (e.g. it'll have each of the `--acount flags listed one after another, along with the 3 instances of the `--deterministic` flag).

I'm guessing this isn't what we want. This was broken already, but I'm mentioning in case it relates to the test failure.

Reply:
This could most easily be fixed by renaming the first case as `BASE_GANACHE_ARGS` or something like that.

Reply:
Oh, that makes sense.

Reply:
Yeah, that could be another PR

Reply:
This has been addressed in #6970